src/NuGet.Clients/NuGet.CommandLine/MsBuildToolset.cs

```diff
@@ -65,6 +65,8 @@ public Version ParsedVersion
             }
         }
 
+        public bool IsValid => Path != null;
```

Comment:
what if Path is empty string?

Reply:
should we do a file/folder exist check instead?

Reply:
No codepath would end up doing that right now, but that's a good point. I will change it.

Reply:
@rohit21agrawal No need for a folder exist check because that's done when this path is evaluated.

Reply:
Double checked it. That is already validated